### PR TITLE
Remove modules from schemas

### PR DESCRIPTION
**Pull request by ssbarnea**
We will no longer check module names and arguments as part of playbook and tasks schemas as these features are greatly dependent of local configuration and they do not scale well.

To be clear there is no JSON validator that can cope with a full schema that checks modules, even if we would generate one.

We will recommend users to use one of the editors that can make use of https://github.com/ansible/ansible-language-server

These schemas ar still valid but you should be aware of their limitations.

### CODE REVIEW

**Review by ziegenberg — APPROVED** [review]
LGTM

File: tox.ini [hunk 1]
```diff
@@ -20,6 +20,7 @@ commands =
   # we use npm/mocha to perform testing instead of pytest, mainly because
   # ajv-cli does not allow passing options we need.
   # {envpython} -m pytest
+  npm run test
```
Comment (webknjaz) [inline tox.ini:23]: Why is this included with the schema removal?

Comment (ssbarnea) [reply tox.ini:23]: Safety. I wanted to see that testing passes. Less than ideal but with current state of affairs...
